backend/src/main/java/ch/puzzle/okr/service/validation/ActionValidationService.java

```diff
@@ -0,0 +1,32 @@
+package ch.puzzle.okr.service.validation;
+
+import ch.puzzle.okr.models.Action;
+import ch.puzzle.okr.repository.ActionRepository;
+import ch.puzzle.okr.service.persistence.ActionPersistenceService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ActionValidationService extends ValidationBase<Action, Long, ActionRepository, ActionPersistenceService> {
+
+    public ActionValidationService(ActionPersistenceService actionPersistenceService) {
+        super(actionPersistenceService);
+    }
```

Comment:
maybe implement method validate on get by key result id, use keyResultValidation service to verify existence of key result